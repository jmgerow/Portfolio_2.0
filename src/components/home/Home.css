.skyline-photo {
    /* width: 100%; */
    margin-top: 10px;
    /* height: 200px; */
    /* margin-bottom: -100px; */
}

#intro {
    /* color: white; */
    font-size: 25px;
}