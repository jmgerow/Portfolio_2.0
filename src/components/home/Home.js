import React, { Component } from 'react';
import { Link } from 'react-router-dom';

import skyline from '../../images/chicago.jpg';

import Homenav from '../home-nav/Homenav';

import './Home.css';

class Home extends Component {
    constructor() {
        super();

        this.state = {
            title: "Justin Gerow"
        }
    }

    render() {
        return (
            <div className="body">
                <div className="banner">
                    <img src={skyline} alt="skyline" className="responsive-img skyline-photo"></img>
                    <div className="container">
                        <div className="row center headline blue-grey-text text-darken-1">
                            <p className="header col s12 light" id="intro">Welcome to my portfolio. This site contains a collection
                            of my works, info about me, and a contact form to leave a note. Thank you for visiting!</p>
                        </div>
                    </div>
                </div>
                <Homenav />
            </div>
        );
    }
}

export default Home;