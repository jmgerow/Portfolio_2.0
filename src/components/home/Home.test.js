import React from 'react';
import { shallow } from 'enzyme';
import Home from './Home';

describe('Home', () => {
    const home = shallow(<Home />)
  
    it('renders without crashing', () => {
      expect(home).toMatchSnapshot();
    });
 
    it('renders homenav on homepage', () => {
        expect(home.find('Homenav').exists()).toBe(true);
    });
  });