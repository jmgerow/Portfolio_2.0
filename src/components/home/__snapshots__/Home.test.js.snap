// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders without crashing 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Home />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          className="banner"
        >
          <img
            alt="skyline"
            className="responsive-img skyline-photo"
            src="chicago.jpg"
          />
          <div
            className="container"
          >
            <div
              className="row center headline blue-grey-text text-darken-1"
            >
              <p
                className="header col s12 light"
                id="intro"
              >
                Welcome to my portfolio. This site contains a collection of my works, info about me, and a contact form to leave a note. Thank you for visiting!
              </p>
            </div>
          </div>
        </div>,
        <Homenav />,
      ],
      "className": "body",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <img
              alt="skyline"
              className="responsive-img skyline-photo"
              src="chicago.jpg"
            />,
            <div
              className="container"
            >
              <div
                className="row center headline blue-grey-text text-darken-1"
              >
                <p
                  className="header col s12 light"
                  id="intro"
                >
                  Welcome to my portfolio. This site contains a collection of my works, info about me, and a contact form to leave a note. Thank you for visiting!
                </p>
              </div>
            </div>,
          ],
          "className": "banner",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "alt": "skyline",
              "className": "responsive-img skyline-photo",
              "src": "chicago.jpg",
            },
            "ref": null,
            "rendered": null,
            "type": "img",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <div
                className="row center headline blue-grey-text text-darken-1"
              >
                <p
                  className="header col s12 light"
                  id="intro"
                >
                  Welcome to my portfolio. This site contains a collection of my works, info about me, and a contact form to leave a note. Thank you for visiting!
                </p>
              </div>,
              "className": "container",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <p
                  className="header col s12 light"
                  id="intro"
                >
                  Welcome to my portfolio. This site contains a collection of my works, info about me, and a contact form to leave a note. Thank you for visiting!
                </p>,
                "className": "row center headline blue-grey-text text-darken-1",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Welcome to my portfolio. This site contains a collection of my works, info about me, and a contact form to leave a note. Thank you for visiting!",
                  "className": "header col s12 light",
                  "id": "intro",
                },
                "ref": null,
                "rendered": "Welcome to my portfolio. This site contains a collection of my works, info about me, and a contact form to leave a note. Thank you for visiting!",
                "type": "p",
              },
              "type": "div",
            },
            "type": "div",
          },
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="banner"
          >
            <img
              alt="skyline"
              className="responsive-img skyline-photo"
              src="chicago.jpg"
            />
            <div
              className="container"
            >
              <div
                className="row center headline blue-grey-text text-darken-1"
              >
                <p
                  className="header col s12 light"
                  id="intro"
                >
                  Welcome to my portfolio. This site contains a collection of my works, info about me, and a contact form to leave a note. Thank you for visiting!
                </p>
              </div>
            </div>
          </div>,
          <Homenav />,
        ],
        "className": "body",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <img
                alt="skyline"
                className="responsive-img skyline-photo"
                src="chicago.jpg"
              />,
              <div
                className="container"
              >
                <div
                  className="row center headline blue-grey-text text-darken-1"
                >
                  <p
                    className="header col s12 light"
                    id="intro"
                  >
                    Welcome to my portfolio. This site contains a collection of my works, info about me, and a contact form to leave a note. Thank you for visiting!
                  </p>
                </div>
              </div>,
            ],
            "className": "banner",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "alt": "skyline",
                "className": "responsive-img skyline-photo",
                "src": "chicago.jpg",
              },
              "ref": null,
              "rendered": null,
              "type": "img",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <div
                  className="row center headline blue-grey-text text-darken-1"
                >
                  <p
                    className="header col s12 light"
                    id="intro"
                  >
                    Welcome to my portfolio. This site contains a collection of my works, info about me, and a contact form to leave a note. Thank you for visiting!
                  </p>
                </div>,
                "className": "container",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <p
                    className="header col s12 light"
                    id="intro"
                  >
                    Welcome to my portfolio. This site contains a collection of my works, info about me, and a contact form to leave a note. Thank you for visiting!
                  </p>,
                  "className": "row center headline blue-grey-text text-darken-1",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Welcome to my portfolio. This site contains a collection of my works, info about me, and a contact form to leave a note. Thank you for visiting!",
                    "className": "header col s12 light",
                    "id": "intro",
                  },
                  "ref": null,
                  "rendered": "Welcome to my portfolio. This site contains a collection of my works, info about me, and a contact form to leave a note. Thank you for visiting!",
                  "type": "p",
                },
                "type": "div",
              },
              "type": "div",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "disableLifecycleMethods": true,
  },
}
`;
